Dockerfile

```diff
@@ -48,7 +48,8 @@ RUN apt-get update \
     libxmlsec1-dev \
     libgeos-dev \
     python3-enchant \
-  && locale-gen en_US.UTF-8
+  && locale-gen en_US.UTF-8 \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
Noice!

We could do this after every layer with an `apt-get update`,
 (https://github.com/dependabot/dependabot-core/pull/3398#issuecomment-816854547 says ~30MB of potential savings).